### PR TITLE
README.md: add simple example

### PR DESCRIPTION
**Pull request by TotallyGamerJet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,42 @@
...
+
+	"github.com/ebitengine/purego"
+)
+
+func getSystemLibrary() string {
+	switch runtime.GOOS {
+	case "darwin":
+		return "/usr/lib/libSystem.B.dylib"
+	case "linux":
+		return "libc.so.6"
```
Comment (hajimehoshi) [inline README.md:23]: Do we need `.6`?

Comment (TotallyGamerJet) [reply README.md:23]: On Ubuntu where I tested, yes

Comment (TotallyGamerJet) [reply README.md:23]: Also [this](https://stackoverflow.com/questions/6841311/using-dlopen-to-get-handle-of-libc-memory-allocation-functions) StackOverflow issue says libc.so is a linker script not an elf file.

Comment (hajimehoshi) [reply README.md:23]: You mean `dlopen("libc.so")` without `.6` did not work?

Comment (hajimehoshi) [reply README.md:23]: I remembered that there is an environment that such a symbolic link doesn't exist. NVM

**Review by hajimehoshi — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (TotallyGamerJet) [reply README.md:23]: Without the .6 it fails on Ubuntu

**Pull request merged**